src/components/projects/create-wp-form/create-wp-form/create-wp-form.tsx

```diff
@@ -41,24 +44,57 @@ const CreateWPFormView: React.FC<CreateWPFormViewProps> = ({
                 <Row>
                   <Form.Group as={Col}>
                     <Form.Label htmlFor='wp-name'>Work Package Name</Form.Label>
-                    <Form.Control id='wp-name' name='wpName' type='text' required></Form.Control>
+                    <Form.Control
+                      id='wp-name'
+                      name='name'
+                      type='text'
+                      onChange={(e: any) => name.setter(e.target.value)}
+                      required></Form.Control>
                   </Form.Group>
                 </Row>
                 <Row>
                   <Form.Group as={Col}>
                     <Form.Label htmlFor='project-wbs-num'>Project WBS Number</Form.Label>
-                    <Form.Control id='project-wbs-num' name='wbsNum' type='text' required></Form.Control>
+                    <Form.Control
+                      id='project-wbs-num'
+                      name='wbsNum'
+                      type='text'
+                      onChange={(e: any) => wbsNum.setter(e.target.value)}
+                      required></Form.Control>
+                  </Form.Group>
+                  <Form.Group as={Col}>
+                    <Form.Label htmlFor='crId'>Change Request ID</Form.Label>
+                    <Form.Control
+                      id='crId'
+                      name='crId'
+                      type='number'
+                      min={1}
+                      onChange={(e: any) => crId.setter(e.target.value)}
+                      required></Form.Control>
                   </Form.Group>
                 </Row>
                 <Row>
                   <Form.Group as={Col}>
                     <Form.Label htmlFor='start-date'>Start Date</Form.Label>
-                    <Form.Control id='start-date' name='startDate' aria-label={'start date input'} type='date' required></Form.Control>
+                    <Form.Control
+                      id='start-date'
+                      name='startDate'
+                      aria-label={'start date input'}
+                      type='date'
+                      onChange={(e: any) => startDate.setter(e.target.value)}
+                      required></Form.Control>
                   </Form.Group>
                   <Form.Group as={Col}>
                     <Form.Label htmlFor='duration'>Duration</Form.Label>
                     <InputGroup>
-                      <Form.Control id='duration' name='duration' aria-label={'duration'} type='number' min={0} required></Form.Control>
+                      <Form.Control
+                        id='duration'
+                        name='duration'
+                        aria-label={'duration'}
+                        type='number'
+                        min={0}
+                        onChange={(e: any) => duration.setter(e.target.value)}
```

Comment:
For all of these I believe the `e: any` isn't necessary since react provides the types, but for the number inputs you might want to parse the values so it's not setting them as strings. I'm not sure about this though and I haven't tested it.

Reply:
Right now, my `handleSubmit` expects all the data to come in as strings, and it seems to work so far. I know we have plans to switch to using react forms in the future, so I'm just gonna leave it alone since it works for now and keep the number inputs as strings.